templates/ts/source/ui.tsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
+import React, {FC} from 'react';
 import {Text} from 'ink';
 
-const App = ({name = 'Stranger'}) => (
+const App: FC<{name?: string}> = ({name = 'Stranger'}) => (
```

Comment:
Could you separate that into an interface?

```ts
interface Props {
  name: string | undefined
}

const App: FC<Props> = ...
```

Reply:
Sure.